_studio/mfx_lib/shared/src/mfx_h264_encode_vaapi.cpp

```diff
@@ -251,6 +249,26 @@ mfxStatus SetRateControl(
     return MFX_ERR_NONE;
 }
 
+mfxStatus SetBrcResetRequired(
+    VADisplay             vaDisplay,
+    VABufferID &          rateParamBuf_id,
+    bool                  isBrcResetRequired)
+{
+    VAEncMiscParameterBuffer *misc_param;
+    VAEncMiscParameterRateControl *rate_param;
+
+    MFX_CHECK_WITH_ASSERT(VA_STATUS_SUCCESS == vaMapBuffer(vaDisplay,
```

Comment:
please, don't put function call inside macro, it could lead to double function calls. see [coding guide](https://github.com/Intel-Media-SDK/MediaSDK/blob/master/doc/Coding_guidelines.md#macro)